pages/common/git-restore.md

```diff
@@ -0,0 +1,17 @@
+# git restore
+
+> Restore working tree files. Requires git version 2.23+.
+> See also `git checkout`.
+> More information: <https://git-scm.com/docs/git-restore/>.
+
+- Restore a file that was accidentally deleted:
+
+`git restore {{path/to/file}}`
+
+- Restore a file to an earlier version:
```

Comment:
The other version is not necessarily earlier. It could simply be from a different branch. Maybe say that explicitly.